### PR TITLE
Refactor Image tests add Lombok

### PR DESCRIPTION
**Pull request by melloware**
- Refactor ImageResource
- Remove some Libs not needed for Native Mode
- Added Lombok

cc @nderwin

### CODE REVIEW

File: integration-tests/pom.xml [hunk 1]
```diff
@@ -10,6 +10,7 @@
     <name>Quarkus JasperReports - Integration Tests</name>
     <properties>
         <skipITs>true</skipITs>
+        <lombok.version>1.18.34</lombok.version>
```
Comment (nderwin) [inline integration-tests/pom.xml:13]: Boo.  Down with this abomination.  :-1:

Comment (melloware) [reply integration-tests/pom.xml:13]: Oh man it makes code so much cleaner I don't do a project without it...

Comment (melloware) [reply integration-tests/pom.xml:13]: Setters and getters are boilerplate waste and i hate writing this...

```java
private static final Logger LOG = Logger.getLogger(Application.class);
```

Comment (nderwin) [reply integration-tests/pom.xml:13]: It's the first thing I look for to refactor out.  Worse yet, no lines of code were saved with the logger declaration.  At least it was the JBoss logger, I always use the JUL logger in my projects. :smiling_imp:  Getters are easy, just an IDE shortcut away.

Then there's the comments here - https://github.com/quarkusio/quarkus/discussions/43179

:shrug: c'est la vie

**Pull request merged**